requirements.txt

```
@@ -27,15 +22,16 @@ termcolor
 jsonlines
 unidic-lite
 einops
-flash-attn==2.2.2
+flash-attn==2.5.2 # should really only be in dockerfile. Local env often doesn't have GPUs
 auto-gptq
 fire
 alpaca-eval==0.6
 # for human eval web app
 flask
-vllm 
+vllm @ git+https://github.com/vllm-project/vllm@fbf152d976e655f8734561452a1036e63081ccfd # todo bump to next release for easier install
```

```suggestion
vllm
```

We don't need to pin this anymore.